apps/dotcom/src/components/CursorChatBubble.tsx

```diff
@@ -184,11 +188,14 @@ const CursorChatInput = track(function CursorChatInput({
 		[stopChatting, setValue]
 	)
 
-	const handlePaste = useCallback((e: ClipboardEvent) => {
-		// todo: figure out what's an acceptable / sanitized paste
-		preventDefault(e)
-		e.stopPropagation()
-	}, [])
+	const handlePaste = useCallback(
+		(e: ClipboardEvent) => {
+			const text = e.clipboardData.getData('text/plain')
+			setValue(text)
+			e.stopPropagation()
+		},
+		[setValue]
+	)
```

Comment:
This is triggering the paste twice now, and overriding the entire chat bubble with your paste.

I think we added the max message length since this todo was placed here, so I reckon we can just remove this paste handler, and allow pastes to happen as normal now

```suggestion
	const handlePaste = useCallback(
		(e: ClipboardEvent) => {
			e.stopPropagation()
		},
		[setValue]
	)
```

Reply:
Paste wasn't happening at all when I tried it, I thought maybe we had punted on paste? I'll roll back this, we can address it separately.